src/instructlab/config/init.py

```diff
@@ -72,61 +75,71 @@ def init(
 ):
     """Initializes environment for InstructLab"""
     ensure_storage_directories_exist()
-    clone_taxonomy_repo = True
-    if interactive:
-        if exists(DEFAULTS.CONFIG_FILE):
-            overwrite = click.confirm(
-                f"Found {DEFAULTS.CONFIG_FILE} in the current directory, do you still want to continue?"
+    cfg = None
+    if os.environ.get(DEFAULTS.ILAB_GLOBAL_CONFIG) is not None:
+        model_path = ctx.obj.config.serve.model_path
+        taxonomy_path = ctx.obj.config.generate.taxonomy_path
+        taxonomy_base = ctx.obj.config.generate.taxonomy_base
+        cfg = ctx.obj.config
```

Comment:
It's working for me, I'm going to push an update to this branch now.

Reply:
are these the only fields we care about copying over? It's still not working for me but that might be because I'm editing fields like `num_cpu` since my undersanding is we want to take the entire config as is

Reply:
nvm, I suppose the entire golden config should be getting stored in the `ctx` obj so I'm not sure why my changes aren't getting reflected

Reply:
`ILAB_GLOBAL_CONFIG=config-leseb.yaml ilab config init` works for me, for now I just edited`general.log_level` and it's reflected in the generated config file.

I'm writing test!